<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>缓和曲线计算程序</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>321</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>输入</string>
      </property>
      <widget class="QPushButton" name="btn_calc">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_clear">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>290</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>交点里程(F)   (K+</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_F">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>m)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>圆曲线半径(R)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>转角(a)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>缓和曲线L0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>桩距</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>80</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_R">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>180</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_L">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_D">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_A1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_A3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_A2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>″</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>130</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>′</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>°</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>359</x>
        <y>30</y>
        <width>361</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>输出</string>
      </property>
      <widget class="QListWidget" name="list_widget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>321</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_F</tabstop>
  <tabstop>edit_R</tabstop>
  <tabstop>edit_A1</tabstop>
  <tabstop>edit_A2</tabstop>
  <tabstop>edit_A3</tabstop>
  <tabstop>edit_L</tabstop>
  <tabstop>edit_D</tabstop>
  <tabstop>btn_calc</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>list_widget</tabstop>
 </tabstops>
 <resources>
  <include location="C:/Users/Cabby/Desktop/x.qrc"/>
 </resources>
 <connections/>
</ui>
